mods/ra/weapons/missiles.yaml

```diff
@@ -90,16 +89,15 @@ HellfireAG:
 		Inaccuracy: 128
 		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		RangeLimit: 4c819
+		RangeLimit: 8c819
```

Comment:
Okay, i reread the PR and understand issue 2 - 20% additional range might not be enough. Please consider picking a relative percentage instead of an arbitrary value regardless.